cellprofiler/utilities/hdf5_dict.py

```diff
@@ -25,10 +25,7 @@
 import six.moves.urllib.parse
 
 
-try:
-    buffer  # Python 2
-except NameError:  # Python 3
-    buffer = memoryview
+buffer = memoryview
```

Comment:
This variable existed to provide a common interface for these keywords - any instance of `buffer` in these files should just be changed to `memoryview` and this assignment should be removed entirely

Reply:
will do